Authenticator/src/Authy/Configuration.pm

```diff
@@ -607,7 +607,7 @@ sub cfg_otp_verification_url {
     my ($otp, $id) = @_;
     croak "No OTP specified" unless defined $otp;
     croak "No ID specified" unless defined $id;
-    return sprintf _AUTHY_OTP_VERIFICATION_URL, $otp, $id, (cfg_otp_allow_unregistered_users() ? 'false' : 'true');
+    return sprintf _AUTHY_OTP_VERIFICATION_URL, $otp, $id, (cfg_otp_allow_unregistered_users() ? 'true' : 'false');
```

Comment:
Can you please help me clarify the usage of `force=true` here? 
What would an unregistered user be in this case? 
Isn't it covered always by `force=false`?  https://www.twilio.com/docs/authy/api/one-time-passwords#force-one-time-password-validation-for-unregistered-user

Reply:
I think, but I was not able to verify this, that it allows you to verify a token sent by SMS for example, even if the user has not register yet.
But well I was not able to verify this.
For sure here `false` / `true` are not in the correct order :)

Many thanks @LucasVidal for all your reviews 👍